docs: add nestjs cli change on migration guide & docs on `--strict` (#2403)
nestjs/docs.nestjs.com: content/first-steps.md

$ nest new project-name
```

> > info **Hint** If you want to create a new project with few flags of TypeScript's strict mode enabled, add the option `--strict` on `new` command. We highly recommend that! More on this option [here](./cli/usages).

Reviewer: ```suggestion
> info **Hint* To create a new project with TypeScript's [strict](https://www.typescriptlang.org/tsconfig#strict) mode enabled, pass the `--strict` flag to the `nest new` command. 
```
